front/app/components/ProjectCard/index.tsx

```diff
@@ -64,7 +64,6 @@ const Container = styled(Link)<{ hideDescriptionPreview?: boolean }>`
 
   &.large {
     width: 100%;
-    min-height: 450px;
```

Comment:
The width of the image is always fixed to 575px. If we use `min-height: 450px`, the ratio is not `4:3`, but `575:450 = 3.8:3`